### PR TITLE
Improved qui-devices widget

### PR DESCRIPTION
**Pull request by marmarta**
Improved visuals for qui devices widget. Some improved behavior, in particular "attach to new disposable" and "allow detaching from only one vm if attached to multiple".

Still uses old API.

requires https://github.com/QubesOS/qubes-core-admin-client/pull/277

### CODE REVIEW

**Comment by marmarta** [comment]
PipelineRetryFailed

**Comment by marmarta** [comment]
also, this requires https://github.com/QubesOS/qubes-core-admin-client/pull/277

File: qui/devices/backend.py [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+        Get the qubesadmin.vm.QubesVM object.
+        """
+        return self._vm
+
+    @property
+    def should_be_cleaned_up(self):
+        """
+        VMs that should have the "shut me down when detaching device" option
+        """
+        return self._vm.auto_cleanup
```
Comment (marmarek) [inline qui/devices/backend.py:92]: Not every VM has this attribute, wrap in `getattr()`

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by marmarek** [comment]
The list of "Attach to new disposable qube" look non-intuitive in bold - it suggests it is the "preferred" action, while it isn't necessary the case. Better make the action in separate row (like with "Attach to qube") and actual disposable templates list in non-bold.

**Comment by marmarek** [comment]
When detaching device from a qube, I get two notifications:
- device detached
- device removed

No such issue when attaching (just one notification)

File: qui/devices/backend.py (hunk 1 above)
Comment (marmarta) [reply qui/devices/backend.py]: I don't know why, but Github doesn't show me that I pushed a fix for that, but the code looks fine. So, done.

**Comment by marmarta** [comment]
Should be working now.

**Review by marmarek — APPROVED** [review]

**Pull request merged**